readme.md

```diff
@@ -162,7 +162,8 @@
 - [Lua](https://github.com/LewisJEllis/awesome-lua#readme)
 - [C](https://github.com/aleksandar-todorovic/awesome-c#readme)
 - [C/C++](https://github.com/fffaraz/awesome-cpp#readme) - General-purpose language with a bias toward system programming and embedded, resource-constrained software.
-- [R](https://github.com/qinwf/awesome-R#readme)
+- [R](https://github.com/qinwf/awesome-R#readme) - Functional programming language and environment for statistical computing and graphics.
+	- [Additional Learning Resources](https://github.com/iamericfletcher/R-Learning-Resources)
```

Comment:
Should be just `Learning` and you forgot `#readme`.

Reply:
Hi @sindresorhus - This PR was orphaned by mistake (still learning my way around GitHub). I created a new PR that accounts for your commented items. 

[Pull Request #1882](https://github.com/sindresorhus/awesome/pull/1882)